.storybook/preview-head.html

```diff
@@ -23,161 +30,122 @@
   @font-face {
     font-family: GT Guardian Titlepiece;
     font-weight: 700;
-    src: url(https://assets.guim.co.uk/static/frontend/fonts/guardian-titlepiece/noalts-not-hinted/GTGuardianTitlepiece-Bold.woff2)
-        format('woff2'),
-      url(https://assets.guim.co.uk/static/frontend/fonts/guardian-titlepiece/noalts-not-hinted/GTGuardianTitlepiece-Bold.woff)
-        format('woff'),
-      url(https://assets.guim.co.uk/static/frontend/fonts/guardian-titlepiece/noalts-not-hinted/GTGuardianTitlepiece-Bold.ttf)
-        format('ttf');
+    src: url(/GTGuardianTitlepiece-Bold.woff2) format('woff2'),
```

Comment:
any reason why we're self hosting the fonts now?

Reply:
I don't see a benefit over using the CDN? As it just bloats the project slightly. Unless I'm missing something

Reply:
It was a potential cause of the font rendering issue, suggested by Chromatic support. I will revert this change, we can revisit if issues continue to crop up.